notebooks/text_classification/text_classification_classical_text_explainer.ipynb

```diff
@@ -333,13 +333,20 @@
    "source": [
     "ExplanationDashboard(local_explanantion)"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python (interpret_cpu)",
+   "display_name": "Python 3",
```

Comment:
the metadata for all notebooks should be consistent (eg display name, environment name of kernel) - these changes are ok but then make sure it applies for all notebooks.
Also, I thought in instructions it said to create the interpret_cpu environment, so maybe it's better to keep the old display name?

Reply:
it's in this section on the main page:

2.1 Create and activate conda environment (For CPU): 
    python tools/generate_conda_files.py
    conda env create -n **interpret_cpu** --file=interpret_cpu.yaml
    conda activate interpret_cpu